pkg/controllers/proxy/network_services_controller.go

```diff
@@ -1055,287 +1088,68 @@ func (ln *linuxNetworking) prepareEndpointForDsr(containerID string, endpointIP
 
 	hostNetworkNamespaceHandle, err := netns.Get()
 	if err != nil {
-		return errors.New("Failed to get namespace due to " + err.Error())
+		return fmt.Errorf("failed to get namespace due to %v", err)
```

Comment:
Why not create a new error object here?

Reply:
`fmt.Errorf()` actually returns a new error object, it just makes formatting a bit easier and more intuitive. Furthermore, when I eventually enable the `errorlint` linter and change all of these `%v` references it will automatically handle error wrapping which is the correct way to stack errors in GoLang so that they are comparable.